# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: 8d80a4ba914c
# Full Hash: 8d80a4ba914c6f81430ba1c55471a8d2b3aac60e
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 2
# Description:
#   Bug 1654112 - Create CallWorkerThread which implements AbstractThread for a TaskQueueWrapper. r=ng
#   
#   Ideally TaskQueueWrapper would implement both webrtc::TaskQueueBase and
#   AbstractThread, but webrtc::TaskQueueBase is not refcounted so implementing both
#   in the same class is not possible.
# ==============================================================================

diff -r e0a2e54e4de8 -r 8d80a4ba914c dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Thu Apr 08 11:31:43 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Mon Apr 12 14:49:20 2021 +0200
@@ -7,6 +7,7 @@
 #include "common/browser_logging/CSFLog.h"
 #include "mozilla/media/MediaUtils.h"
 #include "mozilla/Telemetry.h"
+#include "TaskQueueWrapper.h"
 #include "transport/runnable_utils.h"
 #include "WebrtcCallWrapper.h"
 